cluster/gce/gci/node.yaml

```diff
@@ -117,6 +117,25 @@ write_files:
       [Install]
       WantedBy=multi-user.target
 
+  - path: /lib/systemd/system/google-network-daemon.service
```

Comment:
Will google-network-daemon be installed as a systemd service on all COS/GCI images going forward even when Antrea isn't installed or is there a flag somewhere that ensures this is only active when Antrea is enabled.